### DISCUSSION TITLE
Schedule run & Monitor

### DISCUSSION THREAD

**Discussion by lgcyril** [hoppscotch/hoppscotch#3988] [Q&A]
Hello community,

I am new to Hoppscotch and couldn't the answer to my question : how can we Schedule run & Monitor like Postman?
And run in Bitbucket (CI/CD tool)?

**Comment by liyasthomas** [comment]
Does [Hoppscotch CLI](https://docs.hoppscotch.io/documentation/clients/cli) works for your use case?
